### ISSUE TITLE
The dataloader can't find the validation set

### ISSUE THREAD

**Issue by beckynevin** [oceanites/sen12tp#6]
This is likely a change with the [zenodo repo](https://zenodo.org/records/7342060)- I noticed there's training and test data but no validation data.

Thanks and let me know if you want more details!

I'm attempting to run the [MIMO-Unet model](https://github.com/antonbaumann/MIMO-Unet) with the sen12tp dataset. :)

**Comment by oceanites** [comment]
Great, that you want to use the dataset and try the MIMO-Unet model!

For the SEN12TP dataset it is intentional that there is no defined validation split. Only a test set is specified to allow comparing the results for the same ROIs as in the [original paper introducing the dataset](https://link.springer.com/article/10.1007/s41064-023-00238-y). Already in that paper an ensemble is used which was trained with different train/validation set splits.

I would suggest that you just split the provided train set into a train and validation set. In one of the splits I had 1863 scenes for training and 230 for validation.
